fastlane/lib/fastlane/actions/app_store_build_number.rb

```diff
@@ -159,13 +166,14 @@ def self.available_options
 
       def self.output
         [
-          ['LATEST_BUILD_NUMBER', 'The latest build number of either live or testflight version']
+          ['LATEST_BUILD_NUMBER', 'The latest build number of either live or testflight version'],
+          ['LATEST_VERSION', 'The version of the latest build number']
         ]
       end
 
       def self.details
         [
-          "Returns the current build number of either the live or testflight version - it is useful for getting the build_number of the current or ready-for-sale app version, and it also works on non-live testflight version.",
+          "Returns the current build number and version of either the live or testflight version - it is useful for getting the build_number of the current or ready-for-sale app version, and it also works on non-live testflight version.",
```

Comment:
This is not correct then. It only returns the build number (and also makes the version available in `SharedValues`).

Reply:
Good call. I removed my changes to the description strings in both actions. The main purpose of the actions still is to fetch the build number and the fact that the version string is also exported via the `SharedValues` is already contained in the action details.